src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/CallInstruction.cs

```diff
@@ -49,9 +49,9 @@ public static CallInstruction Create(MethodInfo info, ParameterInfo[] parameters
                 return GetArrayAccessor(info, argumentCount);
             }
 
+#if !FEATURE_DLG_INVOKE
             return new MethodInfoCallInstruction(info, argumentCount);
-#if FEATURE_DLG_INVOKE
```

Comment:
It's dead code right now indeed. There are quite a few of these `FEATURE` flags across the interpreter code base. It seems it'd be worth evaluating whether we'll ever want to re-enable these. If not, we should get rid of them. @VSadov, what do you think?